The MinGW version we have in craft does not support C++17 functions,
so we need to work them around until we have a newer MinGW (update is WIP atm)
---

diff --git a/src/filewidgets/kurlnavigator.cpp b/src/filewidgets/kurlnavigator.cpp
index .. 100644
--- a/src/filewidgets/kurlnavigator.cpp
+++ b/src/filewidgets/kurlnavigator.cpp
@@ -722,7 +722,10 @@
         return button->minimumWidth();
     };
     // Check whether buttons must be hidden at all...
+    int requiredButtonWidth = 0;
+    for (const KUrlNavigatorButton *button : std::as_const(m_navButtons)) {
+        requiredButtonWidth += button->minimumWidth();
+    }
-    const int requiredButtonWidth = std::transform_reduce(m_navButtons.cbegin(), m_navButtons.cend(), 0, std::plus<>(), MinWidth);
 
     if (requiredButtonWidth > availableWidth) {
         // At least one button must be hidden. This implies that the
